docs/source/custom_models.mdx

```diff
@@ -326,6 +326,10 @@ model = AutoModelForImageClassification.from_pretrained(
 Note that when browsing the commit history of the model repo on the Hub, there is a button to easily copy the commit
 hash of any commit.
 
+Also note that the Hugging Face team is in the process of running more and more security scanning of uploaded files
```

Comment:
I think it'll read better if we put this after the second sentence in the section. Something like:

All files and code uploaded to the Hub are scanned for malware (refer to the [Hub security](https://huggingface.co/docs/hub/security#malware-scanning) documentation for more information), but you should still review the model code and author to avoid executing malicious code on your machine. Set `trust_remote_code=True` to use a model with custom code:

Reply:
sounds good to me @stevhliu – mind pushing directly on this PR? 🙏